mindsdb/integrations/handlers/strapi_handler/tests/test_strapi_handler.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+from mindsdb.integrations.handlers.strapi_handler.strapi_handler import StrapiHandler
+from mindsdb.api.mysql.mysql_proxy.libs.constants.response_type import RESPONSE_TYPE
+
+
+class StrapiHandlerTest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        connection_data = {
+            'host': 'localhost',
+            'port': '1337',
+            'api_token': 'dcb6fdf6355e2b49fe18ef4ff618b94cadf16bd2d4e6e33f40a61cdfc5ca6896a3ff2e906cb9da3894ffe38a645aa3e941a8aed65dee16064bfda3ce139b3023b187428a12d295bbfc93162c1aca68fd195180919d8757b6562d641682c652655d9b00a152ef191d8c140e8f9e8d9e8a975944be08e56f387c451513235bfa49',
```

Comment:
Let's add some simple string as example, this should be mocked in the new version of tests

Reply:
done